app/assets/javascripts/discourse/templates/components/reviewable-topic-link.hbs

```diff
@@ -6,6 +6,9 @@
   {{else if (has-block)}}
     {{yield}}
   {{else}}
-    <span class="title-text">{{i18n "topic.deleted"}}</span>
+    <span class="title-text">
+      {{i18n "review.topics.deleted"}}
+      <a href='/t/{{reviewable.removed_topic_id}}'>{{i18n "review.topics.original"}}</a>
```

Comment:
This is not safe for subfolder based installations. You could use the `{{get-url}}` helper.

Reply:
Done :ok_hand: